substrate/frame/election-provider-multi-block/src/signed/tests.rs

```diff
@@ -633,4 +640,626 @@ mod e2e {
 			assert_ok!(Submissions::<Runtime>::ensure_killed(0));
 		})
 	}
+
+	#[test]
+	fn after_rejecting_calls_verifier_start_again_if_leader_exists() {
+		ExtBuilder::signed().build_and_execute(|| {
+			roll_to_signed_open();
+			assert_full_snapshot();
+
+			// Submit invalid solution with high score but no pages (will be slashed)
+			{
+				let mut strong_score = mine_full_solution().unwrap().score;
+				strong_score.minimal_stake *= 2;
+				assert_ok!(SignedPallet::register(RuntimeOrigin::signed(99), strong_score));
+				// Don't submit any pages - this will cause it to be slashed
+			}
+
+			// Submit good solution with all pages
+			{
+				let strong_solution = mine_full_solution().unwrap();
```

Comment:
Or similar to the above test, first submit this one, so you can reuse the `strong_solution.score`, and multiply it by two.